
<launch>

  <node name="tb_topic_inject_carter1" pkg="fydp_mapping" type="tb_topic_inject.py" respawn="false" args="carter1"/>
  <node name="tb_topic_inject_carter2" pkg="fydp_mapping" type="tb_topic_inject.py" respawn="false" args="carter2"/>
  <node name="tb_topic_inject_carter3" pkg="fydp_mapping" type="tb_topic_inject.py" respawn="false" args="carter3"/>

</launch>
